### PR TITLE
Remove the numerical prefix from the file name and Fix links

### LINKED ISSUE #23: Some internal references need to be fixed
In some cases, the references make use of full URLs that point to the localhost. These types of references cause hyperlinks to point to the http://www.localhost:8000/def/current/, which is not the desired behaviour.
image

Some other references implement relative URLs and point to the old name of the course folders, which included a numerical value at the beginning. These references need to be updated.
image
image

### PR DESCRIPTION
**Pull request by jaguarfi**
# Description

In order to fix some of the broken links, the numerical prefix was removed from the file name. This change will help to add new chapters and to avoid renaming all the links that would be affected by having to change the numerical prefix in the file names. This commit also fixes some links that hard-coded the domain to `localhost:8000` and some other links that were broken.

Fixes: #23

# Testing

I used the search function of VSCode to find the links, and I fixed the references one by one while I navigated the course in the **localhost**. I basically tested all the links manually.

# Is it [Done](https://wiki.aalto.fi/display/EDIT/Definition+of+Done)?

*Clean up your git commit history before submitting the pull request!*

- [ ] I (developer) have created unit tests and the tests pass
- [ ] I (developer) have created functional tests (Selenium tests) if applicable
- [X] I (developer) have tested the changes manually
- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

### CODE REVIEW

File: active_elements/introduction.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ elements in the A+ frontend requires that the feedback is wrapped in
 a ``<div id="feedback">`` element.
 
 The active element RST directives, ``ae-input`` and ``ae-output``, are documented in the
-`a-plus-rst-tools <https://github.com/Aalto-LeTech/a-plus-rst-tools#6-active-element-input>`_.
+`a-plus-rst-tools <https://git.io/JJ0XS>`_.
```
Comment (raphendyr) [inline active_elements/introduction.rst]: Why a short link?

Comment (jaguarfi) [reply active_elements/introduction.rst]: It is more concise?

Comment (raphendyr) [reply active_elements/introduction.rst]: It also obfuscates the destination, thus the reader doesn't klnow where the link will lead to. Short links are only useful when people need to manually write them to somewhere (e.g. when printed on paper).

Comment (jaguarfi) [reply active_elements/introduction.rst]: mmmm I thought that having short links might help us to organize the document. But, yes you are right maybe having short links are not a good idea :slightly_smiling_face: I will change it.

File: lti/configuration.rst [hunk 2]
```diff
@@ -18,8 +18,7 @@ in the site before adding any related course content. The sitewide
 configuration of the LTI service contains the key and secret obtained from the
 Tool Provider.
 
-The A+ site administrator accesses the admin interface in the URL path
-`/admin <//localhost:8000/admin>`_.
+The A+ site administrator accesses the admin interface.
```
Comment (markkuriekkinen) [inline lti/configuration.rst]: This paragraph could still show the URL path `/admin` since not everyone knows it.

**Review by markkuriekkinen — COMMENTED** [review]
Some of the changes in this PR seem to collide with #28. Which one should be merged first?

File: lti/configuration.rst (hunk 2 above)
Comment (jaguarfi) [reply lti/configuration.rst]: I will keep the link, but it will point to the https://plus.cs.aalto.fi/admin/ as the manual is now online

**Comment by jaguarfi** [comment]
You can merge this one first. I will resolve any merging conflict in the other PRs

**Review by markkuriekkinen — DISMISSED** [review]
This is OK. I merged the Rubyric PR #25 before looking at the new updates here and now there is a git conflict in the Rubyric chapter.

File: rubyric/index.rst [hunk 3]
```diff
@@ -4,8 +4,7 @@ Module 6 - Rubyric
...
 .. toctree::
-  01_introduction
-  02_getting_started
-  03_rubrics_and_reviews
-  04_roles_and_groups
-  example
+  introduction
+  getting_started
+  rubrics_and_reviews
+  roles_and_groups
```
Comment (markkuriekkinen) [inline rubyric/index.rst:10]: Was the `example` chapter removed by accident?

**Review by markkuriekkinen — COMMENTED** [review]
I assume that the Rubyric example vanished in the git rebase by accident.

File: rubyric/index.rst (hunk 3 above)
Comment (jaguarfi) [reply rubyric/index.rst:10]: Sorry about that. Now should be ready.

**Review by markkuriekkinen — APPROVED** [review]

**Pull request merged**